test/unit/controllers/todo.controller.test.ts

```diff
@@ -0,0 +1,130 @@
+import { expect } from '@loopback/testlab';
+import { TodoController } from '../../../src/controllers';
+import { TodoRepository } from '../../../src/repositories';
+import * as sinon from 'sinon';
+import { Todo } from '../../../src/models/index';
+
+describe('TodoController', () => {
+  let todoRepo: TodoRepository;
+
+  // Method stubs
+  let create: sinon.SinonStub;
+  let findById: sinon.SinonStub;
+  let find: sinon.SinonStub;
+  let replaceById: sinon.SinonStub;
+  let updateById: sinon.SinonStub;
+  let deleteById: sinon.SinonStub;
+
+  // Test fixtures
+  let controller: TodoController;
+  let todo: Todo;
+  let todoWithId: Todo;
+  let changedTodo: Todo;
+  let todoList: Todo[];
+
+  const noError = 'No error was thrown!';
+
+  describe('createTodo', () => {
+    beforeEach(resetRepositories);
+    it('creates a Todo', async () => {
+      expect(await controller.createTodo(todo)).to.eql(todoWithId);
+      sinon.assert.calledWith(create, todo);
+    });
+
+    it('throws if the payload is missing a title', async () => {
+      const payload = new Todo({
+        desc: 'Some description'
+      });
```

Comment:
So what if we add another required property later, and the code verifying required properties will check that new property before the missing "title"? This acceptance test will incorrectly fail in such case.

I think a more robust solution is to use a data builder to create a valid Todo instance data, and then remove only the `title` property.